src/extensions/extensionInterface.ts

```diff
@@ -419,10 +437,25 @@ export const sortingSelectionReducer: Reducer<SortingSelection, SortingSelection
     else if (action.type === 'Set') {
         return action.state
     }
+    else if (action.type === 'ToggleApplyMerges') {
+        return adjustSelectedUnitIdsBasedOnMerges({
```

Comment:
When we turn on 'apply merges' we want to make sure that we deselect any units that are no longer going to be visible (because they are not representatives of their merge groups)